packages/smooth_app/lib/data_models/product_query_model.dart

```diff
@@ -42,26 +43,33 @@ class ProductQueryModel with ChangeNotifier {
   String? get loadingError => _loadingError;
   LoadingStatus get loadingStatus => _loadingStatus;
 
+  /// Sets the translation for meta category "All".
+  void setTranslationForAll(final String translationForAll) =>
+      categories[_CATEGORY_ALL] = translationForAll;
```

Comment:
It's been only on that specific screen, from the start.
With that you get the top right "filter" action that lets you filter by category. And `_CATEGORY_ALL` is a local meta category (the default one) that says "all foods" (that obviously all foods implicitly belong to).

I think we should remove this filter because:
* it's not consistent with "product list page"s (e.g. history, user lists) - or instead we should put the same filter in "product list page"s too
* I don't think it's used (perhaps we should put some sentry code for stats)
* it slows down the page load, as we need to parse _all_ the products before displaying anything in order to get the categories right. Another solution would be to `async`ly load the filter categories while displaying the products, that's possible.